Vagrantfile

```diff
@@ -160,6 +160,7 @@ end
 VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.box_check_update = false
+    config.vbguest.auto_update = false
```

Comment:
@abhinandanpb: Yes, that's right. However, setting `config.vbguest.auto_update = false` is only required in the presence of this plugin. The line I've written above sets this only if vbguest is installed.